### ISSUE TITLE
Suddenly cannot run gulp through Jenkins as of today

### ISSUE THREAD

**Issue by pford68** [gulpjs/gulp#1647]
I am posting this issue here because apparently changes have been made recently:  e.g., the need to install gulp-cli globally instead of gulp.  (This isn't necessarily new, but I have been using Gulp for over two years now, so I may be behind the times.)  Our Jenkins builds have been working for weeks, if not months, then suddenly today, out-of-the-blue, I started getting errors when installing a new module ("dateformat was not found").  A number of people online suggested updating Gulp, that Gulp was the culprit, as well as clean out the node modules and running ``npm install``.  I updated npm and gulp...found the instructions about installing gulp-cli globally and uninstalling the old global Gulp...then my problems really began.

Basically when I try to run a build, Jenkins claims it completed successfully, but the entire UI, built with Gulp, was skipped.  The console reports:

``Unsupported gulp version undefined``

I have the following versions:
* gulp-cli:  1.2.1
* gulp (local) 3.9.1
* npm: 3.8.9
* node: 4.4.0

Curiously I have installed all of the same versions on my local machine, and it builds just fine. 

Any ideas?

**Comment by pford68** [comment]
Whatever the problem is, it appears not to be Gulp related.  I can build with the same build scripts from the command line on the same machine.

**Comment by patryk-wlaz** [comment]
Did you resolve it? Yes, it is not gulp related, but today I run into same issue - Jenkins was running gulp for two months without problems and today jobs keep failing. In the same time I can run gulp as Jenkins user in cmd.
